### PR TITLE
Add Types Declaration

### PR DESCRIPTION
**Pull request by kirevdokimov**
Thank you for the library. Here are types so it's easy to add to typescript projects.

### CODE REVIEW

File: yarn.lock
Comment (stevinz) [inline yarn.lock:1]: This project is published to npm, this file should probably be removed?

File: package.json [hunk 1]
```diff
@@ -42,7 +42,11 @@
   },
   "devDependencies": {
     "rollup": "^2.58.0",
```
Comment (stevinz) [inline package.json:44]: rollup-plugin-dts@5.3.0 requires rollup@"^3.0.0", this should probably be updated to match?

**Review by stevinz — COMMENTED** [review]
Thanks for adding this, it is a good idea.

File: yarn.lock
Comment (kirevdokimov) [reply yarn.lock:1]: Ok

File: package.json (hunk 1 above)
Comment (kirevdokimov) [reply package.json:44]: It works with a current version of rollup, I see no issues here

**Comment by stevinz** [comment]
Looks good, thanks again!

**Pull request merged**